### PR TITLE
Replaced project with a useful warning

### PR DESCRIPTION
**Pull request by jhol**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,266 +1,24 @@
...
-* [puppetlabs-puppetdb](https://github.com/simp/puppetlabs-puppetdb)
-* [puppetlabs-mysql](https://github.com/simp/puppetlabs-mysql)
-* [puppetlabs-java](https://github.com/simp/puppetlabs-java)
-* [puppet-gpasswd](https://github.com/simp/puppet-gpasswd)
-* [augeasproviders_sysctl](https://github.com/simp/augeasproviders_sysctl)
-* [puppet-datacat](https://github.com/simp/puppet-datacat)
-* [puppetlabs-java_ks](https://github.com/simp/puppetlabs-java_ks)
-* [puppet-memcached](https://github.com/simp/puppet-memcached)
+You really should quit your job and find something you can do to make the world
+a better place.
```
Comment (bryonglodencissp) [inline README.md:24]: 1084248904

File: README.md [hunk 2]
```diff
@@ -1,266 +1,24 @@
...
-Please be aware that this project is in the process of transitioning legacy
-code into GitHub. It may take quite some time before we have a fully functional
-system available for immediate download.
+ * https://prism-break.org/en/ - PRISM BREAK
 
-We appreciate both your interest and your patience during this period.
+# To the NSA
 
-You can follow our progress at our [JIRA](https://simp-project.atlassian.net) page.
+Pretty simple really: *Stop spying on us*
```
Comment (bryonglodencissp) [inline README.md:19]: 1084248864

File: README.md [hunk 3]
```diff
@@ -1,266 +1,24 @@
...
+ * http://bit.ly/1I926oj - Rep Grayson, NSA Surveillance: American As Apple Spy 
 
-# **MIGRATION IN PROGRESS**
+For those interested in finding ways to protect themselves from unwarranted
+surveillance through the use of encryption technology.
 
-Please be aware that this project is in the process of transitioning legacy
-code into GitHub. It may take quite some time before we have a fully functional
-system available for immediate download.
+ * https://prism-break.org/en/ - PRISM BREAK
```
Comment (bryonglodencissp) [inline README.md:15]: 1084248781

**Comment by bryonglodencissp** [comment]
1084248946

**Comment by jhol** [comment]
@JLatNSA  - why don't you like my pull request?

**Comment by titdent** [comment]
I'm really sorry I did not really know the program and to learn and I jessaye its not really what its function is as for what purpose is the application I apologize and I shook my google translation because I 'm french and vrait not the contrary I like your program because you seem to you i know and what I did to tell you that I did not like your actual application please please me

Envoyé depuis ma tablette Samsung

-------- Message d'origine --------
De : jhol notifications@github.com 
Date : 07/17/2015  3:22 PM  (GMT-05:00) 
À : NationalSecurityAgency/SIMP SIMP@noreply.github.com 
Objet : Re: [SIMP] Replaced project with a useful warning (#2) 

@JLatNSA - why don't you like my pull request?

**Comment by glamrock** [comment]
Unsurprising response from the open source community... :100:

**Pull request closed without merging**